app/client/cypress/e2e/Regression/ServerSide/QueryPane/DSDocs_Spec.ts

```
@@ -7,18 +7,21 @@ import {
 import { DataSourceKVP } from "../../../../support/Pages/DataSources";
+import { PluginActionForm } from "../../../../support/Pages/PluginActionForm";
 
 let dsName: any;
+let pluginActionForm = new PluginActionForm();
 
 describe(
   "Check datasource doc links",
   { tags: ["@tag.Datasource", "@tag.Git", "@tag.AccessControl"] },
   function () {
-    it("1. Verify Postgres documentation opens", function () {
+    it.only("1. Verify Postgres documentation opens", function () {
```

Biome

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)